Fixed parsing error - quotes

surfingbytes opened the pull request:
There is a double quoted string, which contains unescaped double quotes, so it doesn't work. Changed the string double quotes to single quotes.

stratus-ss commented:
thanks for the PR. I will pull it down and run the tests that I have. This looks pretty straight forward.

Only one question, why are you printing and logging to the file? In general the only time you should be printing to the screen is when the log file cannot be open and you still need to communicate to the end user as such. This isn't intended to be watched as it's going so I would prefer not to do to much stdout printing

surfingbytes commented:
I added the print because when I run the script, it just "did nothing" and looked stuck. So I added that print so there is some ongoing info in the console => doesn't look stuck.
